docusaurus/docs/user-docs/settings/introduction.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Introduction to General Settings
+description: Reviewing the audit logs in Strapi.
+sidebar_position: 2
+---
+
+The ![Settings icon](/img/assets/icons/settings.svg) _Settings_ section in the main navigation of the admin panel comprises of all the required set-up information that determines how an administrator interact with and manage their Strapi application. It is divided into 4 sub sections:
+
+- _Global Settings_
```

Comment:
I'm not sure we need this list here 🤔 It's not wrong, it just does not feel like the way other pages are organized IMHO. I would suggest you use the following structure:
1. Keep the first sentence ("The settings section … manage their Strapi application"
2. Insert the screenshot.
3. Simply introduce the table with a sentence that mixes the current second sentence of the introduction with the current sentence above the table. Something like: "The following table list the 4 Settings sub-sections, available settings in each sub-section, and corresponding links on this user guide to access detailed information:"

Again, your suggestion is not not wrong (apart from the fact that we prefer to use "the following" and never use "above" or "below" in the docs), it just feels, IMHO, a bit less consistent with other introductory pages in the User Guide, hence my suggestion.
What do you think? 😊